src/stm32f1/pwm_reg.hpp

```diff
@@ -0,0 +1,70 @@
+// Copyright 2024 Khalil Estell
```

Comment:
Throw this file into an anonymous namespace in the pwm.cpp file. This pattern of separating the registers and source is only useful if we want to expose these registers to other files.